incubator/nfs-pv-provider/templates/nfs-server-pvc.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{ .Values.server.persistentVolume.name }}"
```

Comment:
Name should use the fullname template.

Reply:
:100: understanding, this is sort of what I imagine you'd use demo label for.

This is actually my first K8S contribution, I've never written any original chart before!

Writing your suggestions in now